rest-client/src/test/java/org/sagebionetworks/bridge/sdk/rest/UserSessionInterceptorTest.java

```diff
@@ -0,0 +1,155 @@
+package org.sagebionetworks.bridge.sdk.rest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.doReturn;
+
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import org.sagebionetworks.bridge.sdk.Tests;
+import org.sagebionetworks.bridge.sdk.rest.model.SignIn;
+import org.sagebionetworks.bridge.sdk.rest.model.UserSessionInfo;
+
+import com.google.common.collect.Lists;
+
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import okhttp3.HttpUrl;
+import okhttp3.Interceptor.Chain;
+import okio.Buffer;
+import okhttp3.MediaType;
+import okhttp3.Protocol;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Request.class, Response.class, Chain.class, HttpUrl.class, ResponseBody.class,
+    Response.Builder.class, Buffer.class})
+public class UserSessionInterceptorTest {
+    
+    private SignIn signIn;
+    private UserSessionInfo userSessionInfo;
+    
+    @Spy
+    UserSessionInterceptor interceptor = new UserSessionInterceptor();
+    
+    @Mock
+    Chain chain;
+    
+    @Mock
+    Request request;
+    
+    @Mock
```

Comment:
I hadn't used the @Mock annotation before, really makes the setup a lot cleaner!